cmd/serve/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const port = ":8081"
```

Comment:
nit: should this be a constant or a command line argument, like the directory path, defaulting to 8081?

Reply:
I wouldn't worry too much about the current structure of the code. This is me basically pumping out code when I get the chance so it's not the cleanest and should mainly be used as a reference point as a state of the union for this effort :laughing: